jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -172,3 +172,12 @@ a {
     color: $cool-grey;
   }
 }
+
+// Utilities
+.u-no-padding--bottom {
+  padding-bottom: 0;
+}
+
+.u-no-margin--bottom {
+  margin-bottom: 0;
```

Comment:
You might want to add a media query here. If I remember correctly. The cols add margin bottom to space things out on small screens.

Reply:
When used on `col-*` classes yes but not relevant for this use case.